pycontrails/datalib/spire.py

```diff
@@ -737,3 +754,663 @@ def _downsample_flight(
 
     # reset original index and return
     return resampled.reset_index(drop=True)
+
+
+class ValidateTrajectoryHandler:
+    """
+    Evaluates a trajectory and identifies if it violates any verification rules.
+
+    <LINK HERE TO HOSTED REFERENCE EXAMPLE(S)>.
+    """
+
+    CRUISE_ROCD_THRESHOLD_FPS = 4.2  # 4.2 ft/sec ~= 250 ft/min
```

Comment:
yep, absolutely.  It is OK if I keep it as FPS within the boundaries of the handler? It will just save having to refactor broadly.

What I'll do is pull this value in from `constants.py`, but still do it as a class variable, so that an implementer can monkey-patch the handler if they want.